rainbow-me/rainbow
# Migrate segment library to latest v2

Fixes TEAM1-38
Figma link (if any):

## What changed (plus any additional context for devs)
- Bumped @segment/analytics-react-native library to latest v2 version
- Updated AsyncStorage library and moved to a new package name for that library
`@react0native-community/async-storage` -> `@react-native-async-storage/async-storage`

## What to test

Prerequisite: Segment dashboard access

- Check if track and identify calls work properly, you can check that in segment debugger for both iOS and Android.
- Check if AsyncStorage still works without any issues, as it should be, which means, checking settings screenm checking if saved wallets are working okay and 

## Final checklist

- [x] Added e2e tests? If not, please specify why – under the hood changes

## Blocked by TEAM1-69
